src/antares/tools/ini_tool.py

```diff
@@ -90,6 +90,19 @@ def ini_dict(self, new_ini_dict: dict[str, dict[str, str]]) -> None:
         self._ini_contents = CustomRawConfigParser()
         self._ini_contents.read_dict(new_ini_dict)
 
+    @property
+    def ini_dict_binding_constraints(self) -> dict[str, dict[str, str]]:
+        return {section: dict(self._ini_contents[section]) for section in self._ini_contents.sections()}
+
+    @ini_dict_binding_constraints.setter
+    def ini_dict_binding_constraints(self, new_ini_dict: dict[str, dict[str, str]]) -> None:
+        """Set INI file contents for binding constraints."""
+        self._ini_contents = CustomRawConfigParser()
+        for index, (section, values) in enumerate(new_ini_dict.items()):
```

Comment:
Could be for index, values in enumerate(new_ini_dict.values()) I think as you don't use section